main.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"flag"
+	"log"
+	"net"
+	"os"
+
+	"google.golang.org/grpc"
+
+	changelogs "github.com/crossplane/crossplane-runtime/apis/changelogs/proto/v1alpha1"
+
+	"github.com/crossplane/changelogs-sidecar/server"
+)
+
+func main() {
+	// get socket file path, allowing user to specify via command line flag
+	socketPath := flag.String("socket-path", "/var/run/changelogs/changelogs.sock", "Path to create a Unix domain socket for change logs gRPC")
+	flag.Parse()
```

Comment:
Is it worth using Kong for consistency with all the other projects we have?

Definitely overkill for one flag, but if we ever need to add more flags it might help avoid breaking changes. iirc the `flag` library uses `-socket-path` (one `-`) style flags, right?

Reply:
From https://pkg.go.dev/flag#hdr-Command_line_flag_syntax, looks like both `-` and `--` are supported:

```
The following forms are permitted:

-flag
--flag   // double dashes are also permitted
-flag=x
-flag x  // non-boolean flags only
```

So I don't think it would be a breaking change to do it later, but I'm happy to include Kong now for consistency sake across the codebase.

Reply:
Kong has been added, I like its simple runtime model for running, error handling, and arguments/flags.